client/object_hash.go

```diff
@@ -74,6 +81,12 @@ func (x PrmObjectHash) GetSession() (*session.Object, error) {
 	return &sess, nil
 }
 
+// IgnoreSession disables auto-session creation.
+// See also [PrmObjectHash.WithinSession].
+func (x *PrmObjectHash) IgnoreSession() {
+	x.isSessionIgnored = true
```

Comment:
It's not enough, session should removed from `meta` as well.

Reply:
Updated